nixos/modules/services/misc/recyclarr.nix

```diff
@@ -0,0 +1,141 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.recyclarr;
+in
+{
+  options.services.recyclarr = {
+    enable = mkEnableOption "recyclarr service";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.recyclarr;
+      defaultText = literalExpression "pkgs.recyclarr";
+      description = "The recyclarr package to use.";
+    };
+
+    command = mkOption {
+      type = types.str;
+      default = "sync";
+      description = "The recyclarr command to run (e.g., sync).";
+    };
+
+    configFile = mkOption {
+      type = types.either types.path types.str;
+      description = lib.mdDoc ''
+        Path to a compatible recyclarr configuration file.
+        The file should be a valid YAML configuration following the recyclarr specification.
+        For detailed configuration options and examples, see the
+        [official configuration reference](https://recyclarr.dev/wiki/yaml/config-reference/).
+      '';
+    };
+
+    workingDir = mkOption {
+      type = types.path;
+      default = "/var/lib/recyclarr";
+      description = "Working directory for recyclarr.";
+    };
+
+    schedule = mkOption {
+      type = types.str;
+      default = "*-*-* 04:00:00"; # Every day at 4 AM
```

Comment:
I'd use `daily` for an easy default instead of the cron-style way.

Reply:
Also, it probably could use the random delay, persistence, etc... options that usually go with a timer.

Reply:
Had those in my `systemd.timers` definition below; I will move them to the option.

Reply:
After searching for examples, I only found [restic](https://github.com/NixOS/nixpkgs/blob/b8228735c965c9dd1356bb8725a34c24b2574844/nixos/modules/services/backup/restic.nix#L141) pursuing this approach. Other services typically expect just the interval, for simplicity, I would suggest keeping the current option definition. WDYT?

Reply:
Sounds good to me.